express-get/server.ts

```diff
@@ -0,0 +1,70 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/countries', async (req, res, next) => {
+  try {
+    const sql = `
+select
+"countries"."name" as "country-name","countries"."countryId" as "countryId",
+count (*) as "cities"
+from
+"cities"
+join "countries" using ("countryId")
+group by "countries"."name","countries"."countryId"
+
+`;
+    const result = await db.query(sql);
+    const countries = result.rows;
+    if (!countries) {
```

Comment:
`countries` is an array, so it will never be falsy. Therefore, this `if` clause will never execute. Just as well leave off this entire check!